File: sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/FileLines20.cs
// Noncompliant {{This file has 20 lines, which is greater than 10 authorized. Split it into smaller files.}}
namespace Tests.Diagnostics
{
    public class FooBar
    {
        public FooBar()
        {
            System
                   .Collections
                   .
                   // WTF
                   Generic

                   .List<int>
                   .
                   // Who writes this kind of code!
                   Enumerator

                   .Equals(null);

            var str = @"
                a
                b";
        }
    }
}

// hello

File: sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/FileLines20.vb
' Noncompliant {{This file has 20 lines, which is greater than 10 authorized. Split it into smaller files.}}
Namespace Tests.Diagnostics
    Class FooBar
        Sub New()
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)
            System.Collections.Generic.List(Of Integer).Enumerator.Equals(Nothing)

            Dim s As String = "a
                b
                c"
        End Sub
    End Class

End Namespace


' Some comment here
